src/main/java/com/jcabi/manifests/MfMap.java

```diff
@@ -40,15 +41,67 @@
  * @since 1.1
  * @see Manifests
  */
-public interface MfMap extends Map<String, String> {
+public interface MfMap {
```

Comment:
@proshin-roman Maybe it's not a good idea to change the public interface of classes that are already in production? Also, what do we gain from this?